lighthouse-core/report/v2/renderer/util.js

```diff
@@ -60,6 +61,16 @@ class Util {
   }
 
   /**
+   * @param {number} ms
+   * @param {number=} granularity Controls how coarse the displayed value is, defaults to 10
+   * @return {string}
+   */
+  static formatMilliseconds(ms, granularity = 10) {
+    const coarseTime = Math.round(ms / granularity) * granularity;
```

Comment:
this rounds to nearest tens not to tenths so we still can't use the opts on `toLocaleString` :/